src/python/library/tritonclient/grpc/__init__.py

```diff
@@ -606,11 +606,7 @@ def get_model_repository_index(self, headers=None, as_json=False):
         except grpc.RpcError as rpc_error:
             raise_error_grpc(rpc_error)
 
-    def load_model(self,
-                   model_name,
-                   headers=None,
-                   config=None,
-                   encoded_files={}):
+    def load_model(self, model_name, headers=None, config=None, files={}):
```

Comment:
Same as Iman's comment for http version of this method. If you switch to `files=None`, you'll need to check `if files` before iterating over it.